compose/config/fields_schema_v2.json

```diff
@@ -0,0 +1,170 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+
+  "type": "object",
+  "properties": {
+    "version": {
+      "enum": [2]
```

Comment:
Minor, but we'll probably want to do semver here, so maybe a string of "2.0" so the version type is consistent when we increment it